New video for scroll-linked animations (#1945)
MicrosoftDocs/edge-developer: microsoft-edge/dev-videos/index.md

-->

<!-- ====================================================================== -->
## Web Platform - Create a scroll-linked animation without JavaScript

_Published on 12 May 2022._

[![Thumbnail image for the scroll-linked animation API video](./images/scroll-linked-animations.png)](https://www.youtube.com/watch?v=Q0nhiHVVnvI)

> Learn about the upcoming CSS scroll-linked animations feature and how it can be used to create a reading progress indicator on a web page without JavaScript.

Reviewer: ```suggestion
Learn about the upcoming _scroll-linked animations_ feature of CSS and how it can be used to create a reading progress indicator on a webpage, without using JavaScript.
```

> This feature is experimentally supported in Microsoft Edge and you can play with it by enabling the "Experimental Web Platform features" flag in `edge://flags`.

Reviewer: Acrolinx (try it & see) would probably suggest breaking up into two sentences; two distinct points are being made.   This is an opportunity to lower the reading grade level.  eg:

This feature is experimentally supported in Microsoft Edge.  To try this feature, enable the "Experimental Web Platform features" flag in `edge://flags`.
```suggestion
This feature is an experimental feature in Microsoft Edge.  To try this feature, go to `edge://flags` and then enable the **Experimental Web Platform features** setting.
```

> You can also play with the demo application shown in the video [here](https://microsoftedge.github.io/Demos/reader/).

Reviewer: change the link text to something descriptive eg [reader] demo.  Avoid [here] as link text per MWSG.  This link is the rendered served-out page at .io - should a link to the source repo be included?.

, see the [rendered reader demo].

> You can also play with the demo application shown in the video [here](https://microsoftedge.github.io/Demos/reader/).

Reviewer: Link to the source repo?

> And learn more about the feature on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/@scroll-timeline).

Reviewer: ```suggestion
To learn more about CSS scroll-linked animations, see [@scroll-timeline](https://developer.mozilla.org/en-US/docs/Web/CSS/@scroll-timeline) at MDN.
```
